### DISCUSSION TITLE
Update 107 for ChromeOS

### DISCUSSION THREAD

**Discussion by synapt0** [3kh0/ext-remover#168] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does anyone know if update 107 or beyond could possibly make the LTBeef or Ingot bookmarklet work again? Or are they gone permanently after 105?

**Comment by ghost** [comment, marked as answer]
No, the patch was on purpose

**Reply by synapt0** [reply]
Got it.
